### ISSUE TITLE
`st2 --version` is printed out to stderr

### ISSUE THREAD

**Issue by armab** [StackStorm/st2#3064]
Interesting findings from the work on [Chef-stackstorm Integration Tests](https://github.com/StackStorm/chef-stackstorm/issues/29).

`st2 --version` result is printed out to `stderr` instead of `stdout`:

```sh
st2 --version 1> stdout.txt

# stdout.txt contents:

```

```sh
st2 --version 2> stderr.txt

# stderr.txt contents:
st2 2.0.1
```

It definitely should output to stdout.

**Comment by nmaludy** [comment]
Looks like this is a bug in the argparse module: https://bugs.python.org/issue18920

Fixed in Python 3.4.

**Comment by emedvedev** [comment]
That's how Python itself does it prior to 3.4, so argparse is just following the convention, pretty much. While `stdout` would be better, I don't think hacking on argparse is worth it.
